docs/src/UsingQAT/TargetingQIR.md

```diff
@@ -0,0 +1,248 @@
+# Targeting QIR
+
+An important aspect is that the default values change with the target
+configuration. The defaults listed in this section are those for the `generic`
+target configuration.
+
+## QAT Configuration
+
+QAT has some high level configuration that dictates what tasks QAT should do and
```

Comment:
This intro here is much more useful then the paragraph above as an introduction to this topic.